app/pages/Auction/RepairBid.js

```diff
@@ -50,17 +51,19 @@ export default class RepairBid extends Component {
           }
         }
         disabled={this.state.isCorrect}
-      />      
+      />
     );
   }
 
   processValue = async (val) => {
     const value = val.match(/[0-9]*\.?[0-9]{0,6}/g)[0];
-    if (value * consensus.COIN > consensus.MAX_MONEY)
+    const dollarydoos = parseFloat(value) * consensus.COIN;
```

Comment:
We still need a type check here. If the user enters for example `"foobar"` then `parseFloat()` returns `NaN` which will throw an error during `setState()`. Suggestion, maybe we need this:

```js
const dollarydoos = parseFloat(value);
if (!Number.isNaN(dollarydoos))
  return;
```

Reply:
`foobar` wouldn't pass the regex.